docker/Dockerfile.deb_debug

```diff
@@ -33,5 +36,8 @@ RUN apt-get update && \
 # net-tools (0.7M): netstat
 # lsof (0.5M): for debugging open socket, file descriptors
 # knot-dnsutils (4.5M): dig/nslookup
+# gdb: for debugging envoy
+# pstack: display a stack trace for each process
```

Comment:
other cmds have size in `()`. i guess that's for tracking the size of the docker image. maybe we should add it for these two as well?

Reply:
Do you know how did we get these numbers?